.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
Why do you use `trusty` ? Does it works with the latest dist?
https://docs.travis-ci.com/user/trusty-to-xenial-migration-guide

Reply:
Trusty is the dist that we're using on all of the other refinery gems. It's fixed the build for 2 other projects, and now this one.

Reply:
I personally have no opinion, what ever fixes the build. :)